mParticle-Rokt/MPKitRokt.m

```diff
@@ -393,7 +382,7 @@ + (NSString *)stringForIdentityType:(MPIdentity)identityType {
             return @"microsoft";
             
         case MPIdentityOther:
-            return @"other";
+            return @"emailsha256";
```

Comment:
Nit - I would add a comment above here that says the reason this isn't `other` - Something like `As of 7/30/2025, customers pass emalsha256 as an "MPIdentityOther"`

Obvious to us right now because we're in the thick of it, but a future developer may not be aware